### DISCUSSION TITLE
Make all function digraphs 1-char

### DISCUSSION THREAD

**Discussion by chunkybanana** [Vyxal/Vyxal#677] [Ideas]
We have a bunch of function digraphs, and a bunch of empty function overload slots. It'd be golfier if the function digraphs were all one byte, which we can do.

**Comment by AMiller42** [comment]
Whoops, it looks like this issue kinda got missed/ignored by everyone. I think deciding which overload to use to replace each diagraph will require some discussion, so I'll convert this to a discussion.

**Reply by chunkybanana** [reply]
(short) list

- `ÞC`
- `¨M`
- `ÞR`
- `ÞZ`
- `Þ↑`
- `Þ↓`
